Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -8268,7 +8268,7 @@ WebGPUEnabled:
 
 WebGPUHDREnabled:
   type: bool
-  status: unstable
```

Comment:
Nit in the commit description: You mention"internal" there, but it's actually "unstable" as seen in the diff.

Need official r? from @tadeuzagallo or @heycam please.